### PR TITLE
Add multiple data option on basic enc dec

### PR DESCRIPTION
**Pull request by asajim**
Multiple data only available when using cbc mode, thus I add options to encrypt and decrypt using 128bits and 256bits key on basic mode.

### CODE REVIEW

File: AESLib.c [hunk 1]
```diff
@@ -64,6 +64,18 @@ void aes128_enc_single(const uint8_t* key, void* data){
 	aes128_enc(data, &ctx);
 }
 
+// encrypt multiple blocks of 128bit data, data_len but be mod 16
+// key is assumed to be 256bit thus 32 uint8_t's
```
Comment (DavyLandman) [inline AESLib.c]: this seems to be a copy-and-paste victim? (the 256bit remark)

File: AESLib.c [hunk 2]
```diff
@@ -64,6 +64,18 @@ void aes128_enc_single(const uint8_t* key, void* data){
 	aes128_enc(data, &ctx);
 }
 
+// encrypt multiple blocks of 128bit data, data_len but be mod 16
```
Comment (DavyLandman) [inline AESLib.c]: `but` -> `must`

File: AESLib.c [hunk 3]
```diff
@@ -179,6 +202,18 @@ void aes128_dec_single(const uint8_t* key, void* data){
 	aes128_dec(data, &ctx);
 }
 
+// decrypt multiple blocks of 128bit data, data_len but be mod 16
+// key is assumed to be 128bit thus 32 uint8_t's
```
Comment (DavyLandman) [inline AESLib.c]: should be 16 `uint_t`'s I think?

File: AESLib.c (hunk 1 above)
Comment (asajim) [reply AESLib.c]: yes, sorry about that

File: AESLib.c (hunk 3 above)
Comment (asajim) [reply AESLib.c]: yes, thank you for the information.

**Pull request merged**